.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+Fixes #. A short summary which serves as a squashed-commit message.
+
+A full description of what's been implemented.
+
+Important changes:
+- [x] ...
```

Comment:
Do these have to be checkboxes instead of a simple bullet list? Checkboxes IMHO only are interesting when creating a `WiP` pull request to show what's not yet done.

Reply:
This is exactly why checkboxes are used here :) For in-progress issues.

Reply:
So far always made PRs when everything from issue has been done and review controls implementation mistakes, testing, style, etc. So no PR with known open points exist.

If we here during review process find the need for changes/additional implementations, who will change the top document? The reviewer or the assigned member based on comments?

Reply:
Because each of them has already been closed :) But they do exist and sometimes we do in-progress PRs with checkboxes some of which are not checked.

It hasn't been specified so far and sometimes it was something like "Ok, I'll change the description" but I'd prefer the implementor to keep the issue up-to-date as they normally know more details.

Reply:
Yeah, different PR handling than experienced so far. But no objection when it's a common agreement.